lua/nvim-lsp-installer/servers/eslint/README.md

```diff
@@ -56,3 +58,4 @@ lsp_installer.on_server_ready(function (server)
     server:setup(opts)
 end)
 ```
+*NOTE*: You will have to install `eslint` package locally to your `node_modules` or you will get `Unable to find ESLint library.`  
```

Comment:
It should be defaulting to a global installation of eslint if it fails to resolve a local installation, maybe this can be rephrased to

```suggestion

*NOTE*: You will have to install the [`eslint` package](https://www.npmjs.com/package/eslint) either locally or globally for the server to run successfully.
```